packages/main/test/specs/ColorPalettePopover.spec.js

```diff
@@ -79,7 +79,9 @@ describe("ColorPalette interactions", () => {
 		await defaultButton.keys("ArrowUp");
 		await firstRecentColorsElement.keys("ArrowUp");
 
-		assert.ok(await moreColorsButton.getProperty("focused"),  "Check if more colors button is focused");
```

Comment:
The both tests do different things,  are we changing the behaviour here, hence the test?

Reply:
Yes, since before the `selected` state feature, what this test did was - it just adds the default color in the recent colors, and when re-opening the palette popover the focus was once again on the **default** button, and after pressing 'ArrowUp' two times, the navigation jumps to -> Recent Colors -> More Colors.

While with our change, the focus now as we discussed will be onto the `selected` item upon opening the Popover leading to different behavior.